### PR TITLE
feat(intake-formats): Add Digital Shadows SearchLight.

### PR DESCRIPTION
**Pull request by PierrickV**

### CODE REVIEW

File: Digital Shadows/digital_shadows_searchlight/_meta/fields.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+event.id:
```
Comment (gbossert) [inline Digital Shadows/digital_shadows_searchlight/_meta/fields.yml:1]: It already exists an ECS field with the exact same name: `event.id`.
If you want to parse the Digital Shadow event id, you could store it in a dedicated field such as `digital-shadows.event.id`

File: Digital Shadows/digital_shadows_searchlight/_meta/fields.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ name: event.id
+ type: keyword
+ description: Event UUID in Digital Shadows SearchLight
+
+event.reason:
```
Comment (gbossert) [inline Digital Shadows/digital_shadows_searchlight/_meta/fields.yml:6]: Every fields declared in this file must be considered as new fields brought by this intake. However, the `event.reason` already exists in ECS. You can remove its declaration from this file.

File: Digital Shadows/digital_shadows_searchlight/ingest/parser.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: digital_shadows_searchlight
```
Comment (gbossert) [inline Digital Shadows/digital_shadows_searchlight/ingest/parser.yml:1]: ECS categorization fields are missing: https://www.elastic.co/guide/en/ecs/current/ecs-category-field-values-reference.html

File: Digital Shadows/digital_shadows_searchlight/ingest/parser.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  field_extraction:
+    actions:
+      - set:
+          event.id: "{{json_event.message.id}}"
+          event.reason: "{{json_event.message.classification}}"
+          event.risk_score: "{{json_event.message.get('risk-assessment').get('risk-level')}}"
+          event.category: "{{json_event.message.get('risk-factors')}}"
+          rule.name: "{{json_event.message.title}}"
+          rule.description: "{{json_event.message.description}}"
+          related.hosts: "{{json_event.message.assets}}"
```
Comment (gbossert) [inline Digital Shadows/digital_shadows_searchlight/ingest/parser.yml:20]: This field doesn't match with the format expected in ECS

File: Digital Shadows/README.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Digital Shadows SearchLight
+
+## Description
+
+Your security team is busy. Digital Shadows SearchLight removes the noise of Threat Intelligence, making it easier to identify what is important so you can make better decisions faster.
```
Comment (gbossert) [inline Digital Shadows/README.md:5]: Include an additional sentence to highlight the benefits of enabling this parser

File: Digital Shadows/README.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# Digital Shadows SearchLight
+
+## Description
```
Comment (gbossert) [inline Digital Shadows/README.md:3]: The logo of Digital Shadows must have a transparent background

File: Digital Shadows/_meta/manifest.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+uuid: 0eed2fe6-1965-4632-ab4f-562779b8fb3c
```
Comment (gbossert) [inline Digital Shadows/_meta/manifest.yml:1]: You must specify the data sources brought by this intake

**Review by gbossert — CHANGES_REQUESTED** [review]
Thank you for this new intake.

File: Digital Shadows/_meta/manifest.yml [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+name: digital_shadows_searchlight
+slug: digital_shadows_searchlight
+description: >-
+   Your security team is busy. Digital Shadows SearchLight removes the noise of Threat Intelligence, making it easier to identify what is important so you can make better decisions faster. 
+data_sources:
+   SearchLight: Helps you minimize risks by detecting data loss, securing your online brand, and reducing your attack surface.
```
Comment (gbossert) [inline Digital Shadows/_meta/manifest.yml:7]: SearchLight is not a data source, please pick one of the official data source

Comment (gbossert) [reply Digital Shadows/_meta/manifest.yml:7]: Data_sources should be set for the technology no on the editor level

File: Digital Shadows/digital_shadows_searchlight/_meta/manifest.yml [hunk 9]
```diff
@@ -0,0 +1,5 @@
+uuid: ccf942fe-c839-42be-a081-5c3f946e80f5
+name: digital_shadows_searchlight
+slug: digital_shadows_searchlight
+description: >-
+   Your security team is busy. Digital Shadows SearchLight removes the noise of Threat Intelligence, making it easier to identify what is important so you can make better decisions faster.
```
Comment (gbossert) [inline Digital Shadows/digital_shadows_searchlight/_meta/manifest.yml:5]: This description must includes the sentence that details how SEKOIA.IO can leverage this integration to improve the security of its customers.

File: Digital Shadows/digital_shadows_searchlight/_meta/manifest.yml [hunk 10]
```diff
@@ -0,0 +1,5 @@
+uuid: ccf942fe-c839-42be-a081-5c3f946e80f5
```
Comment (gbossert) [inline Digital Shadows/digital_shadows_searchlight/_meta/manifest.yml:1]: the icon of the data format must have a transparent background

File: Digital Shadows/digital_shadows_searchlight/ingest/parser.yml [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+        properties:
+            input_field: '{{original.message}}'
+            output_field: message
+    name: json_event
+  - name: field_extraction
+stages:
+  field_extraction:
+    actions:
+      - set:
+          event.id: "{{json_event.message.id}}"
```
Comment (gbossert) [inline Digital Shadows/digital_shadows_searchlight/ingest/parser.yml:14]: not sure you should override this field. @otetard can you confirm this field is automatically set by SEKOIA

**Review by gbossert — CHANGES_REQUESTED** [review]

File: Digital Shadows/digital_shadows_searchlight/ingest/parser.yml (hunk 11 above)
Comment (PierrickV) [reply Digital Shadows/digital_shadows_searchlight/ingest/parser.yml:14]: Replaced by `digital_shadows_searchlight.event.id`

**Review by gbossert — APPROVED** [review]

**Pull request merged**